
image_load_path: /home/quintinumi/project/camera_calibration_img/